### PR TITLE
parse info out of PR comment

### PR DESCRIPTION
**Pull request by ericphanson**
The "description" part won't work until we have https://github.com/JuliaRegistries/Registrator.jl/pull/349, but everything should work (🤞). The following is for testing:

- Registering package: TestPackage
- Repository: https://github.com/ericphanson/DoesNotExist.jl
- Version: v0.1.0
- Description: This is a test description.
- Commit: 976c26fc6165d5501fb765048fd18d2e30181a94
- Git reference: v0.1.0
- Release notes:

> This is my awesome package!
> My release notes have multiple lines.

### CODE REVIEW

**Comment by DilumAluthge** [comment]
You'll need to checkout the repo.

**Comment by ericphanson** [comment]
> You'll need to checkout the repo.

Good point. I don't want to though :(. General is pretty big and I just want that one script... I just copied it into the workflow for now. I guess checking out the repo isn't that bad though.

**Comment by ericphanson** [comment]
I think it will be kind of unmaintainable if I just have one giant script so I just checked out the repo instead.

**Comment by ericphanson** [comment]
Ok, this is good to go from my end. In Slack:
Screenshot 2021-08-31 at 22 38 04

File: .ci/parse_env.jl [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+            <!-- END RELEASE NOTES -->
+            """)
+
+using Test
+p = parse_env(env; verbose=false)
+
+@testset "parse_env" begin
+    @test p.text == "Fastnet: This is a test description.\n\nRelease notes:\n> Fastnet is a Julia package that allows very fast (linear-time) simulation of discrete-state dynamical processes on networks, such as commonly studied models of epidemics\n\nRegistration: https://github.com/JuliaRegistries/General/pull/43765\nRepository:   https://github.com/bridgewalker/Fastnet.jl\n"
+    @test p.pr_url == "https://github.com/JuliaRegistries/General/pull/43765"
+end
```
Comment (ericphanson) [inline .ci/parse_env.jl:74]: This is a bit odd since it runs every time the action runs. Then again, it's like CI for the script 😄

**Comment by DilumAluthge** [comment]
I haven't reviewed all of the code in the PR, but the output in Slackmlooks good.

**Pull request merged**